packages/@aws-cdk/core/lib/bundling.ts

```diff
@@ -79,6 +79,41 @@ export interface BundlingOptions {
    * @experimental
    */
   readonly local?: ILocalBundling;
+
+  /**
+   * The type of output that this bundling operation is producing.
+   *
+   * @default BundlingOutput.AUTO_DISCOVER
+   *
+   * @experimental
+   */
+  readonly output?: BundlingOutput;
```

Comment:
Rename to `outputType` ("output" is too overloaded in this context).

Reply:
done, the type remains `BundlingOutput`?